### ISSUE TITLE
random_seed cannot actually be set when creating a new study

### ISSUE THREAD

**Issue by baperry2** [sherpa-ai/sherpa#114]
https://github.com/sherpa-ai/sherpa/blob/ff6466e99717983f9f394ba72b63f17343e32bdc/sherpa/core.py#L84

This functionality shows up in the documentation of the function, but not the implementation.

Ideally, the ability to set the random seed when creating a new study would be added to enable reproducibility. If not, it should be removed from the documentation to avoid confusion.

**Comment by theisjendal** [comment]
I agree. It would be very nice to have reproducible experiments. Also if an error occurs it would be easier to debug as well.
